File: src/test/testdata/assetsInCode/findUsages/FindUsagesWithTags1.java
import com.badlogic.gdx.graphics.Color;
import com.badlogic.gdx.scenes.scene2d.ui.Skin;
import com.gmail.blueboxware.libgdxplugin.annotations.GDXAssets;

class FindUsages1 {

  @GDXAssets(skinFiles = "src\\findUsages/findUsagesWithTags1.skin")
  static Skin staticSkin;

  void f() {
    staticSkin.get("red", Color.class);
    new com.badlogic.gdx.utils.Json().addClassTag("Tag", com.badlogic.gdx.scenes.scene2d.ui.TextButton.TextButtonStyle.class)
  }

}

File: src/test/testdata/assetsInCode/findUsages/FindUsagesWithTags1.kt
import com.badlogic.gdx.graphics.Color
import com.badlogic.gdx.scenes.scene2d.ui.Skin
import com.gmail.blueboxware.libgdxplugin.annotations.GDXAssets

@GDXAssets(atlasFiles = arrayOf(""), skinFiles = arrayOf("src\\findUsages/findUsagesWithTags1.skin"))
val s: Skin = Skin()

fun f() {
  val c = s.get("red", Color::class.java)
}